eclass/gnustep-base.eclass

```diff
@@ -171,36 +171,42 @@ egnustep_env() {
 	die "gnustep-make not installed!"
 }
 
+# @FUNCTION: egnustep_make
+# @DESCRIPTION:
 # Make utilizing GNUstep Makefiles
 egnustep_make() {
 	if [[ -f ./Makefile || -f ./makefile || -f ./GNUmakefile ]] ; then
-		emake ${*} "${GS_ENV[@]}" all || die "package make failed"
+		emake ${*} "${GS_ENV[@]}" all
```

Comment:
Without having checked the details, I say that `${*}` for the function's parameters looks suspicious and should likely be replaced by `"$@"`.